### DISCUSSION TITLE
First things...

### DISCUSSION THREAD

**Discussion by becarpenter** [becarpenter/book6#1] [Q&A]
We have several things to decide before writing anything:
 - Outline of contents, including the order (top down? bottom up?)
 - Which open source license?
 - How to we share the writing?
 - Markdow, yes, but how do we do diagrams?
 - What else?

**Comment by joaolucasmacedo** [comment]
I more for a bottom up strategy.

**Comment by becarpenter** [comment]
Yes, but things have to fit some sort of overall logic, so I think that the list of contents is important too. What I have noticed in the small parts I have already drafted is that certain topics emerge as *missing* - for example, yesterday I wrote the sentence "However, making the last two settings work smoothly can be challenging and is discussed at [TBD](tbd)." Each time something like that appears, we should probably add a new item to the list of contents. Maybe we should open an issue for each case like this, and look for a volunteer to write that new topic?

**Comment by timchown** [comment]
> On 5 Sep 2022, at 21:36, Brian E Carpenter ***@***.***> wrote:
> 
> Yes, but things have to fit some sort of overall logic, so I think that the list of contents is important too. What I have noticed in the small parts I have already drafted is that certain topics emerge as missing - for example, yesterday I wrote the sentence "However, making the last two settings work smoothly can be challenging and is discussed at TBD." Each time something like that appears, we should probably add a new item to the list of contents. Maybe we should open an issue for each case like this, and look for a volunteer to write that new topic?

That sounds good, and is a good way for people to pick up small chunks of text to work on when they have a spare moment.

Tim
